07.ls_object/ls.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'optparse'
+require_relative 'outputter'
+
+options = ARGV.getopts('a', 'r', 'l')
+file_names = Dir.glob('*', options['a'] ? File::FNM_DOTMATCH : 0)
+file_names = file_names.reverse if options['r']
+outputter = Outputter.new(file_names)
+options['l'] ? outputter.long_option_output : outputter.output
```

Comment:
```suggestion
outputter = Outputter.new(file_names)
outputter.output(long: options['l'])
```

こんな感じに書けた方がよりスッキリしそうです。

Reply:
キーワード引数ですね👀
チェリー本5.4.3 読み直して修正してみました。